mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.td

```diff
@@ -487,6 +487,23 @@ def SparsificationAndBufferization : Pass<"sparsification-and-bufferization", "M
            "Enable i32 indexing into vectors (for efficient gather/scatter)">,
     Option<"enableGPULibgen", "enable-gpu-libgen", "bool", "false",
            "Enable GPU acceleration by means of direct library calls">,
+    Option<"parallelization", "parallelization-strategy", "mlir::SparseParallelizationStrategy",
```

Comment:
nit, since you typically have emit < parallel in the order in which they appear
can you either swap these, or make the parameter order the same

I like the flags to read in the same order everwhere